src/v2/Components/ArtworkFilter/index.tsx

```diff
@@ -275,10 +275,11 @@ export const BaseArtworkFilter: React.FC<
 
           <Spacer mb={2} />
 
-          {enableCreateAlert && savedSearchEntity && (
+          {!!savedSearchEntity && (
```

Comment:
@dimatretyak I think we could do a little more "separation of concerns" here. Instead of rendering both filter pills and the "create alert" button when a `savedSearchEntity` is passed, what do you think about separating them so that rendering filter pills is based on a `renderFilterPills` prop and the button on a `renderCreateAlertButton` prop?

Reply:
In theory, this is a good idea 🤔. But it seems to me that such changes will only complicate our logic and we will often have to write the duplicate code.

Reply:
If it doesn't bother you, can you give an example of the benefits that we will get if we add these changes?

Reply:
PR #10391 (simpler solution)
PR #10391 (a more complex solution with duplication of information in some places, but it gives us more flexibility)